garage/tf/samplers/batch_sampler.py

```diff
@@ -143,6 +147,7 @@ def process_samples(self, itr, paths):
         logger.record_tabular('AverageDiscountedReturn',
                               average_discounted_return)
         logger.record_tabular('AverageReturn', np.mean(undiscounted_returns))
+        logger.record_tabular('eprewmean', np.mean(self.eprewmean))
```

Comment:
if we are going to log this, we should use our own naming convention (e.g. `EpisodeRewardMean`). I think there are already mappings between our names and baseline names in the benchmark scripts.

Reply:
Yes there are mappings for names. I used this naming to view the regression in tensorboard. I agree we should follow our naming convention. It would be better if we can move this extra info to somewhere else in the future. Because it only serves for benchmark.

Reply:
Perhaps just put them under "Extras/" or "Info/"?